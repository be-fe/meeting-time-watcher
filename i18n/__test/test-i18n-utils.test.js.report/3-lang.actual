{}
  list : []
  map : {}
    i18n : "还有"
    i18n cn : "还有"
    i18n en : "  left."
    i18n not exist : "还有"