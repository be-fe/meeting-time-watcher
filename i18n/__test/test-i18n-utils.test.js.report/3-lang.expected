{}
  list : []
  map : {}
    i18n : "还有"
    i18n en_US : "  left."
    i18n zh_CN : "还有"